client/src/components/ui/navigationbar/NavDrawer.svelte

```diff
@@ -0,0 +1,78 @@
+<script>
+    import NavQuery from './NavQuery.svelte';
+
+    export let show = false;
+    export let navItems;
+
+    let selected;
+</script>
+
+<nav class:show={!show}>
+  <div>
+        {#each navItems as item} 
+            <a href={`#/${item.href}`} on:click={() => {selected = item.href}} class={selected == item.href ? "selected" : ""}>
+                {item.label}
+            </a>
+      {/each}
+  </div>
+  <form method="POST" action="/auth/logout">
+      <input type="submit" value="Logout" />
+  </form>
+</nav>
+<style>
+    :global(body) {
+        padding: 0;
+    }
```

Comment:
```suggestion
@import url(../../../pages/global.css);
```
Already in `global.css`